test/library/draft/DistributedMap/v3/DistributedMap.chpl

```diff
@@ -0,0 +1,360 @@
+
+module DistributedMap {
+    private use Map;
+    private use ChainTable;
+    private use HaltWrappers;
+    private use CyclicDist;
+    private use IO;
+    private use Reflection;
+    private use ChapelLocks;
+
+    record distributedMap {
```

Comment:
For the sake of supporting aggregation later, this will need to be a record-wrapped class (otherwise you're going to end up copying the contents of the distributed map and failing to see updates in the original).  But I think just leaving a note about that rather than actually implementing it is sufficient.  I just don't want us to forget that and rediscover it if we start the eventual implementation from this version